pkg/provisioner/controller.go

```diff
@@ -470,7 +470,42 @@ func (c *obcController) handleProvisionClaim(key string, obc *v1alpha1.ObjectBuc
 		return fmt.Errorf("error getting reference to OBC: %v", err)
 	}
 
-	// create Secret and ConfigMap
+	// Create auth Secret and bucket info ConfigMap
+	// If the operator crashes before the OB is created, provisioning will happen again, and a
```

Comment:
Just for the confirmation(to understand the code flow for me), OBC creation handled by the provisioner by Grant()/Provision(). Then request return back to library and then it creates cm or secret or both and before completing OB operator crashed, so we have dangling secrets or cm or both. When operator comes back it receives again on OBC claim, so operator need to hanlde Grant()/Provision() request gracefully without failing from its end.

Can we have similar situation Deletion() of OBC claim?

Reply:
I think your understanding sounds correct. I will update the comment to clarify "provisioning" on this line should be "Provision()/Grant()". 

I'm not quite sure I understand what you mean about deletion though...